templates/new/web_main.py.txt

```diff
@@ -17,9 +17,7 @@ RUNNERS = [
 
 ASSEMBLIES = [
     Assembler("baseline")
-        .set_validator(InputValidator())
-        .set_estimator(Baseline())
-        .set_visualiser(ReportGenerator())
```

Comment:
Are we making it not generate a HTML report by default? Or is this accidental?

Reply:
This was on purpose. I want to consolidate the functionality and get a robust version out while the more experimental features get refined in a different branch.